launcher/src/components/UI/the-staking/DisplayValidators.vue

```diff
@@ -752,6 +802,14 @@ export default {
     hideBDialog() {
       this.bDialogVisible = false;
     },
+    hideWDialog() {
+      this.riskWarning = false;
```

Comment:
if you hide the dialog you also need to reset the button.
after i click cancel there is no button to import a key anymore

Reply:
@mabasian